src/post/idea.rs

```diff
@@ -19,7 +19,7 @@ pub struct Idea {
     pub comments: Vec<CommentId>,
 
     // Specific fields
-    pub submissions: Vec<SubmissionId>,
+    pub submissions: Vec<SolutionId>,
```

Comment:
Can the field be renamed to `solutions` or would it be too disruptive? I don't remember where it is even used, to be honest.

Reply:
Okay